solr/core/src/java/org/apache/solr/cloud/api/collections/BackupCmd.java

```diff
@@ -138,6 +138,15 @@ public void call(ClusterState state, ZkNodeProps message, NamedList<Object> resu
 
       backupMgr.writeBackupProperties(backupProperties);
 
+      // It can't be done within aggregateResults call
+      // since endTime is filled later
+      if(backupProperties != null) {
+        // It is safe to extract results here
+        @SuppressWarnings("unchecked")
+        NamedList<Object> response = (NamedList<Object>) results.get("response");
+        response.add("endTime", backupProperties.getEndTime());
```

Comment:
`backupProperties.getEndTime()` would return `null` if it hadn't been filled in later for some reason; that could be null-checked but that would add complexity and so this is good; yet more complicated would be for `backupMgr.writeBackupProperties` to somehow return properties which it added or updated but that would be overcomplex and so this here is good.